RootTools/python/samples/samples_13TeV_RunIIFall17MiniAOD.py

```diff
@@ -266,11 +288,13 @@
 
 # TTbar cross section: NNLO, https://twiki.cern.ch/twiki/bin/view/LHCPhysics/TtbarNNLO (172.5)
 TTJets = kreator.makeMCComponent("TTJets", "/TTJets_TuneCP5_13TeV-amcatnloFXFX-pythia8/RunIIFall17MiniAOD-94X_mc2017_realistic_v10-v1/MINIAODSIM", "CMS", ".*root", 831.76, fracNegWeights=0.319)
+TTJets_SingleLeptonFromTbar = kreator.makeMCComponent("TTJets_SingleLeptonFromTbar", "/TTJets_SingleLeptFromT_TuneCP5_13TeV-madgraphMLM-pythia8/RunIIFall17MiniAOD-94X_mc2017_realistic_v10-v1/MINIAODSIM", "CMS", ".*root", XS_to_be_definied)
```

Comment:
please put in the proper cross section * BR for this sample:  `831.76*(3*0.108)*(1-3*0.108)`.
also, either put this near the `TTJets_SingleLeptonFromT` or put that one near this one - it makes no sense to have the two samples in different places of the file

Reply:
also, you should add the sample to the `TTs` list definition below.